x/gamm/client/cli/cli_test.go

```diff
@@ -78,7 +79,7 @@ func (s *IntegrationTestSuite) TestNewCreatePoolCmd() {
 		newAddr,
 		sdk.NewCoins(sdk.NewInt64Coin(s.cfg.BondDenom, 200000000), sdk.NewInt64Coin("node0token", 20000)), fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
 		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
```

Comment:
Do we also want to deduplicate broadcast mode?

Reply:
thats a good idea, though I wonder if theres a better way we can just get more of these common flags to all be handled, to better mitigate the boilerplate. (E.g. Just appending all the common flags to the args array)

We should probs open an issue for tracking how to improve this